### ISSUE TITLE
Newbie Question: Why this over official https://github.com/slackhq/node-slack-sdk

### ISSUE THREAD

**Issue by jagan-veeraraghavan** [smallwins/slack#47]
I am choosing which node slack sdk to use. Wanted to understand why use this library over the official node-slack.sdk. 
I see this is being actively developed so are there any features available that are missing from the official SDK?

**Comment by brianleroux** [comment]
Hi @jagan-veeraraghavan, definitely active. We use this lib ourselves. Primary difference is this library is stateless, and deliberately thin surface generated from Slacks own docs. 

While it does support the RTM API this lib is more appropriate for bot logic responding to the (also stateless) Events API. It's designed to be fast, tested in both Node and the browser. 

tl;dr If you are using web sockets you may want node-slack or Botkit. If you are building on the Events API or just web hooks integration this library or Slapp might be a better choice.

😅 LMK if that answers your q! Happy to discuss further.

**Comment by mbrevoort** [comment]
+1 it's lightweight. We use it on a lot of production apps as well. Also check out github.com/BeepBoopHQ/slapp which is geared for the Events API 😉

**Comment by brianleroux** [comment]
Closing! LMK if there's any further q's! <3
